content/en/api/tracing/send_trace.md

```diff
@@ -38,7 +38,7 @@ and each span is a dictionary with a `trace_id`, `span_id`, `resource`..
 *   **`duration`** - _required_ The duration of the request in nanoseconds.
 *   **`parent_id`** - _optional_ The span integer ID of the parent span.
 *   **`error`** - _optional_ Set this value to 1 to indicate if an error occured. If an error occurs, you should pass additional information, such as the error message, type and stack information in the `meta` property.
-*   **`meta`** - _optional_ A dictionary of key-value metadata. e.g. tags.
+*   **`meta`** - _optional_ A dictionary of key-value metadata. e.g. tags. NOTE: all keys and values must be strings
```

Comment:
I find the capitalised "NOTE" a bit aggressive. Maybe we can simply rephrase the original sentence, like:
```
**`meta`** - _optional_ A set of key-value metadata. Keys and values must be strings.
```
Or something like that.

You should also add:
```
**`metrics`** - _optional_ A set of key-value metadata. Keys must be strings and values must be 64-bit floating point numbers.
```